app/controllers/admin_outgoing_message_controller.rb

```diff
@@ -33,6 +34,8 @@ def update
                                                  :editor => admin_current_user,
                                                  :old_body => old_body,
                                                  :body => @outgoing_message.body,
+                                                 :old_raw_body => old_raw_body,
+                                                 :raw_body => @outgoing_message.raw_body,
```

Comment:
I'm not sure we should introduce a concept of `raw_body` here.

I think a less complex solution (that essentially mirrors the change to the form) is:

```diff
diff --git a/app/controllers/admin_outgoing_message_controller.rb b/app/controllers/admin_outgoing_message_controller.rb
index c852a7c29..d7629e977 100644
--- a/app/controllers/admin_outgoing_message_controller.rb
+++ b/app/controllers/admin_outgoing_message_controller.rb
@@ -24,7 +24,7 @@ class AdminOutgoingMessageController < AdminController
   end

   def update
-    old_body = @outgoing_message.body
+    old_body = @outgoing_message.raw_body
     old_prominence = @outgoing_message.prominence
     old_prominence_reason = @outgoing_message.prominence_reason
     if @outgoing_message.update_attributes(outgoing_message_params)
@@ -32,7 +32,7 @@ class AdminOutgoingMessageController < AdminController
                                                { :outgoing_message_id => @outgoing_message.id,
                                                  :editor => admin_current_user,
                                                  :old_body => old_body,
-                                                 :body => @outgoing_message.body,
+                                                 :body => @outgoing_message.raw_body,
                                                  :old_prominence => old_prominence,
                                                  :old_prominence_reason => old_prominence_reason,
                                                  :prominence => @outgoing_message.prominence,
```

Reply:
That's nicer to use, thanks!